src/main/play-doc/general/MigrationGuide.md

```diff
@@ -33,18 +33,425 @@ In additional to disabling Elasticsearch via configuration, a service lookup mus
+        service-name  = "my-service"
+        acls          = [
+          {
+            http = {
+              requests = [
+                {
+                  path-beg = "/my-service"
+                }
+              ]
+            }
+          }
+        ]
+      }
+    }
+```
+
+#### Supplying customized HAProxy template configuration
+
+The following services definition will require a customized HAProxy configuration.
+
+The [[Dynamic proxy configuration|DynamicProxyConfiguration]] describes the steps required to generate a customized HAProxy configuration.
+
+Generating a customized HAProxy configuration will rely on the [[ifAcl|DynamicProxyConfiguration#ifAcl]] directive which accepts `system`, `systemVersion`, and `endpointLabel` as input argument.
+
+When declaring bundle settings via `build.sbt`, unless declared explicitly, the `system` and `systemVersion` settings may fallback to a default value as described by [sbt-conductr settings](https://github.com/typesafehub/sbt-conductr#bundle-settings).
+
+
+##### Customized port number
+
+A service may declare an endpoint exposed on a custom port number. Given the service URI `http://:6789/my-service`, the service is accessible from the outside world on HTTP port `6789` under `/my-service` context.
+
+###### Migrating build.sbt
+
+Modify the `Endpoint` declared within the `BundleKeys.endpoints`.
+
+From:
+
+```
+name := "my-bundle"
+BundleKeys.compatibilityVersion := "1.1"
+BundleKeys.endpoints := Map(
+  "endpoint-label" -> Endpoint("http", 0, Set(URI("http://:6789/my-service")))
+)
+
+```
+
+To:
+
+```
+BundleKeys.endpoints := Map(
+  "endpoint-label" -> Endpoint("http", 0, "my-service",
+    RequestAcl(Http("^/my-service".r -> "/"))
+  )
+)
+```
+
+
+###### Migrating bundle.conf
+
+Modify the `endpoints` section declared within the `bundle.conf` as such.
+
+From:
+
+```
+    system             = "my-bundle"
+    systemVersion      = "1.1"
+    endpoints          = {
+      "endpoint-label" = {
+        bind-protocol  = "http"
+        bind-port      = 0
+        services       = ["http://:6789/my-service"]
+      }
+    }
+```
+
+To:
+
+```
+    endpoints         = {
+      "endpoint-label" = {
+        bind-protocol = "http"
+        bind-port     = 0
+        service-name  = "my-service"
+        acls          = [
+          {
+            http = {
+              requests = [
+                {
+                  path-beg = "/my-service"
+                  rewrite = "/"
+                }
+              ]
+            }
+          }
+        ]
+      }
+    }
+```
+
+###### Supplying customized HAProxy configuration
+
+The [[Dynamic proxy configuration|DynamicProxyConfiguration]] describes the steps required to generate a customized HAProxy configuration.
+
+Populate the "Custom Endpoint Configuration" section described in the [[populate custom HAProxy configuration template|DynamicProxyConfiguration#Populate-the-custom-HAProxy-configuration-template]] step as such:
```

Comment:
@markusjura - added link to steps required to supply customized HAProxy configuration